inferred-spans/build.gradle.kts

```
@@ -21,7 +21,7 @@ dependencies {
 
   testAnnotationProcessor("com.google.auto.service:auto-service")
   testCompileOnly("com.google.auto.service:auto-service-annotations")
-  testImplementation("io.opentelemetry.semconv:opentelemetry-semconv-incubating:1.25.0-alpha")
+  testImplementation("io.opentelemetry.semconv:opentelemetry-semconv-incubating:1.26.0-alpha")
```

I think here we should probably just remove the version from this definition in order to align with other modules that get the version from general dependency management.
```suggestion
  testImplementation("io.opentelemetry.semconv:opentelemetry-semconv-incubating")
```

---

For example here: https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/aws-resources/build.gradle.kts#L15